sdk/formrecognizer/Azure.AI.FormRecognizer/src/ClassifyDocumentOperation.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis
+{
+    /// <summary>
+    /// Tracks the status of a long-running operation for classifying documents.
+    /// </summary>
+    public class ClassifyDocumentOperation : Operation<AnalyzeResult>, IOperation<AnalyzeResult>
```

Comment:
Is it possible that properties could be added in the future to the result of an `AnalyzeDocumentOperation` that don't apply to `ClassifyDocumentOperation`? Or to put it another way: Should we have a new `ClassifyResult` (separate from `AnalyzeResult`) to "future-proof" this?

Reply:
That's a good question and it's one of the major open discussions for these new features. Also, there's the point that most properties in `AnalyzeResult` will be empty when using the classify API, so it feels like they should be kept separated. For this first beta we're exposing the models in the same way the service exposes them but this will be a topic of discussion for the next ones.

Reply:
Awesome. Good to know that this is being tracked. Thank you!